coderedcms/templates/coderedcms/pages/web_page.html

```diff
@@ -3,6 +3,7 @@
 
 {% block content_pre_body %}
   {% if self.cover_image %}
+  {% block cover_image %}
```

Comment:
This template doesn't seem to need the block, because it already is within a `block content_pre_body`.